### ISSUE TITLE
[Discussion] Additional reserved words

### ISSUE THREAD

**Issue by ratoaq2** [guessit-io/guessit#302]
```
+-------------------------------------------------------+
+                   GuessIt 2.0.5                       +
+-------------------------------------------------------+
|      Please report any bug or feature request at      |
|     https://github.com/guessit-io/guessit/issues.     |
+-------------------------------------------------------+
```

There are some words that should be detected by guessit:
- `INTERNAL`
- `AVC`
- `DUB` (Dubbed)
- `DIRFIX` (like proper)
- `MULTI`

```
guessit The.Blacklist.S03E03.INTERNAL.720p.HDTV.x264-BATV
For: The.Blacklist.S03E03.INTERNAL.720p.HDTV.x264-BATV
GuessIt found: {
    "title": "The Blacklist", 
    "season": 3, 
    "episode": 3, 
    "episode_title": "INTERNAL", 
    "screen_size": "720p", 
    "format": "HDTV", 
    "video_codec": "h264", 
    "release_group": "BATV", 
    "type": "episode"
}
```

```
guessit "Friends.S10.(2003-2004).BDRip-AVC"
For: Friends.S10.(2003-2004).BDRip-AVC
GuessIt found: {
    "title": "Friends", 
    "season": 10, 
    "year": [
        2003, 
        2004
    ], 
    "format": "BluRay", 
    "release_group": "AVC", 
    "type": "episode"
}
```

```
guessit -t episode "Game.of.Thrones.S05E07.HDTV.720p.DUB.[qqss44].mkv"
For: Game.of.Thrones.S05E07.HDTV.720p.DUB.[qqss44].mkv
GuessIt found: {
    "title": "Game of Thrones", 
    "season": [
        5, 
        44
    ], 
    "episode": 7, 
    "format": "HDTV", 
    "screen_size": "720p", 
    "release_group": "DUB.[qqs", 
    "container": "mkv", 
    "mimetype": "video/x-matroska", 
    "type": "episode"
}
```

**Comment by ratoaq2** [comment]
Just as a reference:
The scene tags:

ALTERNATIVE.CUT, CONVERT, COLORIZED, DC, DIRFIX, DOCU, DUBBED, EXTENDED, FESTIVAL, FINAL, INTERNAL, LIMITED, MULTI, NFOFIX, OM, PPV, PROOFFIX, PROPER, REAL, REMASTERED, READNFO, REPACK, RERIP, SAMPLEFIX, SOURCE.SAMPLE, SUBBED, THEATRICAL, UNCENSORED, UNRATED, UNCUT, and WS

**Comment by ratoaq2** [comment]
Fixed by #420
